esp/src/src/nls/hu/hpcc.ts

```diff
@@ -468,7 +475,23 @@ export = {
     Log: "Log",
     LogAccessType: "Log hozzáférés típusa",
     LogDirectory: "Log könyvtár",
+    LogEventType: "Eseménynapló bejegyzés típus",
     LogFile: "Log fájl",
+    LogFilterComponentsFilterTooltip: "Csak a kiválasztott komponenshez/konténerhez tartozó eseménynapló bejegyzések",
+    LogFilterCustomColumnsTooltip: "A megadott eseménynapló oszlopok bevonása",
+    LogFilterEndDateTooltip: "Naplósorok felvétele az időtartomány végéig",
+    LogFilterEventTypeTooltip: "Csak a kiválasztott tipusú eseménynapló bejegyzásek",
```

Comment:
Typo, the correct one is:
"Csak a kiválasztott tipusú eseménynapló bejegyzések"